pkg/components/components.go

```diff
@@ -2,14 +2,14 @@ package components
 
 import (
 	"fmt"
+	"regexp"
 
 	cnav1alpha1 "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/v1alpha1"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	rbacv1 "k8s.io/api/rbac/v1"
 	extv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
```

Comment:
please keep this empty line.

There are 4 main import groups:
1. Standard library imports
2. Third party imports
3. Own organization imports (i.e. kubevirt imports).
4. Own repo imports

`opv1alpha1 "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/v1alpha1"
	"github.com/kubevirt/cluster-network-addons-operator/pkg/util/k8s"		"github.com/kubevirt/cluster-network-addons-operator/pkg/util/k8s"` belongs to the last group, ans as such should be grouped.

Reply:
btw I see there is a duplicate import `cnav1alpha1` but you don't need to fix it - I'll do it in a separate PR